### PR TITLE
tests workflow

### PR DESCRIPTION
**Pull request by SarahMAmann**
Jira ticket: https://amberdata.atlassian.net/browse/DER-506

REST endpoint unit tests run automatically in GHA on a cron schedule, and will alert slack and opgenie on failures.

### CODE REVIEW

File: .github/workflows/dependabot-security-updates.yml
Comment (SarahMAmann) [inline .github/workflows/dependabot-security-updates.yml:1]: This workflow was broken and prevented the unit tests workflow from running so I just commented it out.

**Review by SarahMAmann — COMMENTED** [review]
Should I delete the tests/__pycache__ folder contents?

File: .github/workflows/dependabot-security-updates.yml
Comment (joanesespanol) [reply .github/workflows/dependabot-security-updates.yml:1]: @adampointer any idea or guidelines here?

**Comment by joanesespanol** [comment]
> Should I delete the tests/**pycache** folder contents?

Yes please!

File: .github/workflows/dependabot-security-updates.yml
Comment (adampointer) [reply .github/workflows/dependabot-security-updates.yml:1]: It does not work on public repositories. We will need to delete this workflow from all public repos.

**Pull request closed without merging**